pylons/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "pylons",
+  "assets": [
+    {
+      "description": "The revenue token for Pylons",
+      "denom_units": [
+        {
+          "denom": "ubedrock",
+          "exponent": 0
+        },
+        {
+          "denom": "rock",
+          "exponent": 6
+        }
+      ],
+      "base": "ubedrock",
+      "name": "Pylons",
+      "display": "rock",
+      "symbol": "ROCK",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/Crouton-Digital/Chain-registry/master/pylons/images/pylons.png"
```

Comment:
Please point to the cosmos' Chain Registry here, not a fork

Reply:
(and for all cases where you have an image)

Reply:
fixed